src/App.tsx

```diff
@@ -16,46 +45,76 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
+  const [sortParams, setSortParams] = useState({
+    property: '',
+    reverse: false,
+  });
+  const preparedGoods = getGoods(goodsFromServer, sortParams);
+
   return (
     <div className="section content">
       <div className="buttons">
         <button
           type="button"
-          className="button is-info is-light"
+          className={cn(
+            'button is-info',
+            { 'is-light': sortParams.property !== 'name' },
+          )}
+          onClick={() => setSortParams({
+            ...sortParams,
+            property: 'name',
+          })}
```

Comment:
it is bad practise to make anonymous arrow functions in jsx.
make handler in component and pass reference here